posthog/api/person.py

```diff
@@ -333,7 +333,7 @@ def delete_property(self, request: request.Request, pk=None, **kwargs) -> respon
             now=datetime.now(),
             sent_at=None,
             event={
-                "event": "$delete_person_property",
+                "event": "$identify",
```

Comment:
maybe do `$set` instead - that we also use for setting person properties from some libraries, we do some more complex logic for `$identify` events, e.g. look at if we should buffer it or not